portfolio/src/main/webapp/script.js

```diff
@@ -41,3 +41,36 @@ function toggleDescription(sectionId) {
   }
   toggleClass(sectionElement, "collapsed");
 }
+
+/**
+ * Change the slide to the current one + n
+*/
+function changeSlide(n, slider) {
+  if (!n || !Number.isInteger(n)) {
+    throw "changeSlide first argument is not an integer.";
+  }
+  let totalSlides = 0;
+  let currentlyDisplayed;
+  for (const elem of slider.children) {
+    const classes = elem.className.split(" ");
+    if (classes.indexOf("slide") !== -1) {
+      if (classes.indexOf("hidden") === -1) {
```

Comment:
I tried `.includes()` but it gave me an error.

Reply:
ok. might be not be supported correctly then. Usually we could transpile the newer JS so it would work on older browsers. But the setup is a bit complicated for now.